### DISCUSSION TITLE
Add template in issues?

### DISCUSSION THREAD

**Discussion by swamper123** [FreeOpcUa/python-opcua#1189] [Ideas]
I don't know where github is hiding this feature:

I know that there are template possibilities, when you open an issue (that there is some text already) to add some text..something like:

Before opening an issue, please read the checklist below and add an ✔️  (use :heavy_check_mark:) behind every point:

- I have a programming issue in the python-opcua library
- I know that questions/ideas/... shall be posted in [Discussions](https://github.com/FreeOpcUa/python-opcua/discussions) or I use the [Gitterchannel](https://gitter.im/FreeOpcUa/python-opcua)
- I will not post unnecessarry Capslock stuff
- I will provide example code if possible
- I read and accept the points above

To help you and us faster, you may want to add some informations about your setup:
- Python Version:
- python-opcua version (master? 0.9?0.83?...):
-  OPC UA Server/Client Software:
- OPC UA Server/Client Hardware:

I know that is another annoying step for issues, but maybe this will safe some time instead of moving them into discussion every time. Also providing informations we may ask anyway is saveing time as well...

![grafik](https://user-images.githubusercontent.com/24408657/101598104-f1b7ec00-39f7-11eb-9eb5-0d3696b8ca71.png)

**Comment by oroulet** [comment]
Done with standarard python template. with just some small changes
